intel-retail/retail-use-cases
# feat: Add gRPC python helm deployment

## PR Checklist



## What are you changing?

Adding helm deployment for grpc python example

## Issue this PR will close

close: #44 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- cd use-cases/grpc_python
- make run-minikube-demo
- kubectl get pods
- See grpc_python pod running

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/retail-use-cases )
